src/usecases/updateWardVisitTotals.js

```diff
@@ -9,10 +12,15 @@ export default ({ getDb }) => async ({ wardId, date }) => {
     const [existingTotal] = res;
 
     if (existingTotal) {
+      const newTotal = existingTotal.total + 1;
       await db.none("UPDATE ward_visit_totals SET total = $2 WHERE id = $1", [
         existingTotal.id,
-        existingTotal.total + 1,
+        newTotal,
       ]);
+
+      const sendVisitsMilestoneNotification = getSendVisitsMilestoneNotification();
+
+      await sendVisitsMilestoneNotification({ numberOfVisits: newTotal });
```

Comment:
I think we could do this asynchronously, I think I'd prefer not to make the app wait for this when it's non-essential. It may not matter as it's the last thing happening in this request but someone could add something later. I'd also like to make sure that a slack error won't cause the request to error.

Reply:
Ah yush, this makes sense! An error in Slack shouldn't affect this usecase as it will be caught in the gateway which then just returns an error key to the usecase. Any improvement ideas or is this okay?

Reply:
Is this the right total? Or is this intentionally looking for 500 visits in a single ward?

Reply:
Ah, ah, ah... 🤦 You're right, this is for a single ward. Good spot! 🕵️‍♂️

Looks like I need to think about this more in my own time.